### ISSUE TITLE
Questions about the training

### ISSUE THREAD

**Issue by forbyme** [haofeixu/gmflow#3]
Hi, nice work!  How did you determine the number of training iterations for each datasets? such as 800K for Things and 200K for Sintel.  Is this the optimal number of iterations?  How did you select and set them?

**Comment by haofeixu** [comment]
Hi, thanks! For Things dataset, we did an ablation on different number of training iterations (Table 2(f) of our paper) and found the performance got gradually improved from 200K to 800K iterations, but we didn't observe more obvious gains when using 800K+ iterations.

For Sintel dataset, the iteration number is set emperically: our 200K model has similar performance with RAFT on Sintel training set and thus we simply choose it.

**Comment by forbyme** [comment]
thanks! How about the Chairs? What does the ''our 200K model has similar performance with RAFT on Sintel training set'' means? Do you determine the number of iterations by  the EPE drop on the validation dataset?

**Comment by haofeixu** [comment]
Since Chairs dataset is relatively smaller and simpler compared with Things, and it is used at the early stage of pre-training, I didn't expect training longer on Chairs would benefit too much to the final performance. Thus I simply set the training iterations on Chairs as 100K, which also has lower EPE than RAFT on Chairs validation set.

For the final Sintel model, I didn't use a validation set. During training on Sintel, the evaluation results on Sintel training set will be reported in our code, which measures how the model fits the Sintel training set. You can find more in previous papers like RAFT or PWC-Net (the results on Sintel training set, which are often reported in the parenthesis).

**Comment by forbyme** [comment]
Thank you for your careful explanation and wish you success in your scientific research

**Comment by haofeixu** [comment]
Thank you so much for your kind words and hope you all the best!
